tests/queries/0_stateless/data_minio/est/metadata/v1.metadata.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Do we need both `v1.metadata.json` and `v2.metadata.json`? Will we test both of them?